#Variable distance

float64 distance           # profundidad a la que está el objeto reconocido
rcnn_live_detector/Prediction prediction     # predicción del objeto reconocido

